src/main/java/cloud/cholewa/game/logic/GameStarter.java

```diff
@@ -56,4 +56,8 @@ public static int setNumberOfPlayers(Scanner scanner) {
 
         return playerList;
     }
+
+    public static int setNumberOfGameRounds() {
```

Comment:
Tutaj mi się nie podoba. Po pierwsze zgodnie z konwencją 'set' powinien ustawiać pole klasy, tutaj powinien być getNumberOfGameRoudns. Po drugie jest to metoda statyczna, publiczna, która zwraca stałą. O wiele zrobić po prostu stałą w GameStarterze - public static final NUMBER_OF_ROUNDS  = 4 i bezpośrednio do niej się odnosić.

Reply:
Poprawiłem... i chyba w pozostałych też trzeba to zrobić tak samo. 
Jeżeli stworzę pole w GameStarter, to nie można korzystać ze statycznych metod. Z kolei tworzenie osobnego pola, z którego korzysta, póki co, jedna metoda, nie za bardzo ma sens na tym etapie